packages/mui-material/src/Select/Select.test.js

```diff
@@ -1271,6 +1271,40 @@ describe('<Select />', () => {
     expect(container.getElementsByClassName(classes.select)[0]).to.toHaveComputedStyle(selectStyle);
   });
 
+  ['standard', 'outlined', 'filled'].forEach((variant) =>
```

Comment:
Let's also add test using `styleOverrides` too, as we slowly want to move to that API in v6

Reply:
Good point. Done.